assets/section-rich-text.css

```diff
@@ -34,6 +34,19 @@
   padding-bottom: 5rem;
 }
 
+@media screen and (min-width: 750px) {
+  .rich-text:not(.rich-text--full-width),
+  .rich-text--full-width.color-background-1 {
+    margin-top: calc(5rem + var(--page-width-margin));
+    margin-bottom: calc(5rem + var(--page-width-margin));
+  }
+
+  .rich-text:not(.color-background-1) {
+    padding-top: calc(5rem + var(--page-width-margin));
+    padding-bottom: calc(5rem + var(--page-width-margin));
+  }
+}
+
```

Comment:
Same here maybe. Since they're right next to each other they could maybe be grouped (media queries).